packages/tupaia-web/src/api/api.ts

```diff
@@ -6,7 +6,8 @@
 import axios from 'axios';
 import FetchError from './fetchError';
 
-export const API_URL = process.env.REACT_APP_TUPAIA_WEB_API_URL || 'http://localhost:8100/api/v1/';
+export const API_URL =
+  import.meta.env.REACT_APP_TUPAIA_WEB_API_URL || 'http://localhost:8100/api/v1/';
 
```

Comment:
I think potentially this should be port 8000, as this is the `web-config-server`, until we switch over to the new server